File: python/nt_toolbox/plot_spectrogram.py
import matplotlib.pyplot as plt
import numpy as np

def plot_spectrogram(S,title):

    S = abs(S[:np.shape(S)[0]/2,:])
    S = np.log(S + 1e-4)
    plt.imshow(S,cmap = plt.get_cmap("jet"), interpolation = "nearest")
    plt.title(title)
    plt.show()

File: julia/NtToolBox/src/plot_spectrogram.jl
function plot_spectrogram(S,titre)
    S = abs(S[1:Int(size(S)[1]/2),:])
    S = log.(S + 1e-4)
    imshow(S, cmap = get_cmap("jet"), interpolation = "nearest")
    title(titre)
end
